### ISSUE TITLE
Broken media link under "Integrations" on homepage

### ISSUE THREAD

**Issue by joshlavroff** [get-alex/alex#353]
### Subject of the issue

On the project homepage (<https://alexjs.com/>) there is a dead media link for <https://raw.githubusercontent.com/get-alex/atom-linter-alex/master/screencast.gif?token=AA5pFhwJyhXtc0a3GdKXaLfWqyIQvN0Mks5V5dJUwA%3D%3D>, which leads to a 404. It appears as the alt text "screen cast"

### Your environment

*   **OS**:  Windows 10 Home v2009
*   **Packages**: N/A 
*   **Env**: Chrome 135.0.7049.115

### Steps to reproduce

Go to <https://alexjs.com/> and scroll to the "Integrations" section

### Expected behaviour

A gif should be displayed

### Actual behaviour

The alt text for the missing gif is displayed

**Comment by utsa-aaa** [comment]
Hi! I’m interested in working on this issue for a class project and would love to contribute a fix. Is this issue still available, or is someone already working on it? I would be happy to take it on if not!
